go/lint.py

```diff
@@ -128,8 +128,20 @@ def parse(line, filename, tool):
         file = file[2:]
 
       if not parts[1].isdigit():
-        row = int(parts[0])
         msg = parts[1]
+
+        try:
+          row = int(parts[0])
+        except ValueError:
+          # In case of missing position, display an error in the first line
```

Comment:
should we show the error next to the cursor, just thinking of larger files where you won't notice the first line.

Reply:
Good idea! You merely have a chance to notice that error even with 100-lines file.

Reply:
Well, actually the problem is from, there is no variable passed from where I might get the current position of the view :thinking: